test/fhirdefs/load.test.ts

```diff
@@ -1,113 +1,59 @@
 import { loadFromPath, loadDependency } from '../../src/fhirdefs/load';
 import { FHIRDefinitions } from '../../src/fhirdefs/FHIRDefinitions';
-import { getResolver } from '../testhelpers/getResolver';
-import { ResolveFn } from '../../src/fhirtypes/ElementDefinition';
+import { Type } from '../../src/utils';
+import { TestFisher } from '../testhelpers';
 import path from 'path';
 import fs from 'fs-extra';
 import tar from 'tar';
 import rp from 'request-promise-native';
 
 describe('#loadFromPath()', () => {
   let defs: FHIRDefinitions;
-  let resolve: ResolveFn;
   beforeAll(() => {
     defs = new FHIRDefinitions();
     loadFromPath(
       path.join(__dirname, '..', 'testhelpers', 'testdefs', 'package'),
       'test#1.1.1',
       defs
     );
-    resolve = getResolver(defs);
-    resolve('Condition');
-    resolve('boolean');
-    resolve('Address');
-    resolve('patient-mothersMaidenName');
-    resolve('allergyintolerance-clinical');
+    // Run the dependency resources through TestFisher to force them into the testhelpers cache
+    const fisher = new TestFisher().withFHIR(defs);
```

Comment:
I guess in the `FHIRDefinitions.test.ts` file you could maybe do something like this to ensure that the definitions you are going to use will be in the cache? But I don't know I don't think it is the cleanest solution, and wouldn't want to hold this up over something that'll only be an issue if we someday switch to R5 or something.

Reply:
I should have read ahead.  Yeah, I noted this as a possible approach when replying to your last comment.